plotting/plot_factory/README.md

```diff
@@ -0,0 +1,49 @@
+# Plot Factory
+
+This directory contains the plot factory, responsible for constructing plot traces and figures to 
+return a DashApp for direct insertion into a web-page.
```

Comment:
Add short definition of plot traces, plot figures and DashApp here for completeness